pkgs/applications/audio/mopidy/mpd.nix

```
@@ -0,0 +1,23 @@
+{ stdenv, python3Packages, mopidy }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "Mopidy-MPD";
+  version = "3.0.0";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "0prjli4352521igcsfcgmk97jmzgbfy4ik8hnli37wgvv252wiac";
+  };
+
+  propagatedBuildInputs = [mopidy];
+
+  # no tests implemented
+  doCheck = false;
```

ensure that it's usable :)
```suggestion
  doCheck = false;
  pythonImportsCheck = [ "mopidy_mpd" ];
```

---

Done